src/Microsoft.WSMan.Management/WsManHelper.cs

```diff
@@ -496,19 +496,13 @@ internal string ProcessInput(IWSManEx wsman, string filepath, string operation,
         internal string GetXmlNs(string resUri)
         {
 
-            string tmpNs = null;
+            string tmpNs = StripParams(resUri);
 
-            if (resUri.ToLowerInvariant().Contains(URI_IPMI) || (resUri.ToLowerInvariant().Contains(URI_WMI)))
-                tmpNs = StripParams(resUri);
-            else
-            {
-                //tmpNs = StripParams(resUri) + ".xsd";
-                //This was reported by Intel as an interop issue. So now we are not appending a .xsd in the end.
-                tmpNs = StripParams(resUri);
-            }
+            //if (!(resUri.ToLowerInvariant().Contains(URI_IPMI) || resUri.ToLowerInvariant().Contains(URI_WMI)))
+            //    tmpNs += ".xsd";
+            //This was reported by Intel as an interop issue. So now we are not appending a .xsd in the end.
```

Comment:
This comment can probably be deleted.

Reply:
OK, I also inlined the `tmpNs` variable. I hope it's OK